modules/core/classes/GalleryStorage/PostgreSqlStorage.class

```diff
@@ -31,6 +31,9 @@ GalleryCoreApi::requireOnce('modules/core/classes/GalleryStorage.class');
  */
 class PostgreSqlStorage extends GalleryStorage {
 
+    /* deprecated dynamic properties in php 8.2 */
+    public $_serverInfo;
```

Comment:
I don't think this is needed.

Reply:
Without the variable declaration unit test 1814 fails with the deprecation warning from php 8.2